### PR TITLE
refactor respository

### PR DESCRIPTION
**Pull request by blankdots**
move repository from elixir-oslo to neicnordic

### CODE REVIEW

File: streaming/out.go [hunk 1]
```diff
@@ -75,6 +76,72 @@ func NewCrypt4GHWriter(writer io.Writer, writerPrivateKey [chacha20poly1305.KeyS
...
+	}
+	_, err = writer.Write(newBinaryHeader)
+	if err != nil {
+		return nil, err
+	}
+	crypt4GHWriter.writer = writer
+	crypt4GHWriter.buffer.Grow(headers.UnencryptedDataSegmentSize)
+	return &crypt4GHWriter, nil
+}
+
```
Comment (jbygdell) [inline streaming/out.go:144]: This function should have a test case

**Review by jbygdell — CHANGES_REQUESTED** [review]

File: streaming/out.go (hunk 1 above)
Comment (blankdots) [reply streaming/out.go:144]: do I add it as part of this PR or another?

Comment (jbygdell) [reply streaming/out.go:144]: The best would be to put this and the corresponding test in a separate PR.
That way this PR don't introduce any real code changes.

Comment (blankdots) [reply streaming/out.go:144]: ah my bad i think i added some changes i made for myself, will fix it

**Review by jbygdell — APPROVED** [review]

**Review by sstli — APPROVED** [review]
This repo move which was agreed upon at the Heilsa all hands meeting. PR contains just name changes from elixir-oslo to neicnordic.

**Review by kjellp — APPROVED** [review]
LGTM

**Comment by blankdots** [comment]
I will merge this and see if we can do a release

**Pull request merged**